Src/CSharpier.Tests/TestFiles/ObnoxiousEdgeCases/ObnoxiousEdgeCases.cst

```diff
@@ -57,10 +52,10 @@ class ClassName
     public static IRemoteAuthenticationBuilder<
         RemoteAuthenticationState,
         RemoteUserAccount
-    > AddAccountClaimsPrincipalFactory<[DynamicallyAccessedMembers(
-            DynamicallyAccessedMemberTypes.PublicConstructors
-        )]
-        TAccountClaimsPrincipalFactory>(
+    > AddAccountClaimsPrincipalFactory<
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)]
```

Comment:
Will this still break if the attribute or its parameters were long enough?

Reply:
Looks like it, I added a test case that breaks